schemas/common_definitions.json

```diff
@@ -193,6 +201,11 @@
           "required": [
             "answer_count"
           ]
+        },
+        {
+          "required": [
+            "type"
```

Comment:
`type` can't be required as most when rules don't define a `type`

Reply:
Isn't this oneOf for the whole block?  I thought it was saying 'you need one of id, meta, answer_count, type', is that wrong?

Reply:
My bad, that'll teach me for commenting on github without viewing the whole file 🤦‍♂️